src/entrypoints/surveys.ts

```diff
@@ -1,9 +1,15 @@
 import { generateSurveys } from '../extensions/surveys'
+import { SurveyEventReceiver } from '../extensions/surveys/survey-event-receiver'
+import { canActivateRepeatedly, hasEvents } from '../extensions/surveys/surveys-utils'
 
 import { window } from '../utils/globals'
 
 if (window) {
+    ;(window as any).__PosthogExtensions__ = (window as any).__Posthog__ || {}
```

Comment:
it upsets me every time i add a new random thing directly onto window
tbh this should probably be `;(window as any).__PosthogExtensions__.surveys.blah`